Dockerfile

```diff
@@ -12,7 +12,7 @@ COPY . .
 RUN mkdir build && \
     cd build && \
     cmake .. \
-        -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+        -DCMAKE_BUILD_TYPE=Release \
```

Comment:
It's smaller, but there is no way to debug a core if the server crashes like in #2305.

We may set up a whole group of images without debugging symbols.

Reply:
Official packages in Debian, Ubuntu, Archlinux... don't include debug symbols.
* Is it possible to reduce the size of Debug symbols? I mean, the binary is just 3MB and debug symbols are 100 MB...
* If the previous point is not possible I think it's a good idea to release 2 Docker images. Keep the current tag for slim version and add debug suffix for the fat one eg:`1.9.2-debug`